solr/webapp/web/WEB-INF/web.xml

```diff
@@ -79,12 +79,16 @@
     <servlet-name>LoadAdminUI</servlet-name>
     <servlet-class>org.apache.solr.servlet.LoadAdminUiServlet</servlet-class>
   </servlet>
-  
+
   <!-- Fullstory additions -->
   <servlet>
     <servlet-name>Statz</servlet-name>
     <servlet-class>org.apache.solr.servlet.StatzServlet</servlet-class>
   </servlet>
+  <servlet>
+    <servlet-name>PrometheusStatz</servlet-name>
+    <servlet-class>org.apache.solr.servlet.PrometheusStatzServlet</servlet-class>
+  </servlet>
```

Comment:
naming nit: I think we should call this "Metrics" rather than "Statz" for consistency with the rest of our usages.  Statz is the old-style.

Reply:
Sunds good to me! Will change